### ISSUE TITLE
GPUs not being used while finetuning

### ISSUE THREAD

**Issue by rosequ** [allenai/bilm-tf#132]
I'm trying to finetune pre-trained ELMo model on Ubuntu 16.04 with 8 core CPUs and a 8G GPU. While finetuning, I see that all the cores are being occupied but the GPU is not used at all. Is this a known issue?

**Comment by rosequ** [comment]
![image](https://user-images.githubusercontent.com/8274617/49103357-1b56ae00-f249-11e8-970d-fbc506f63bfd.png)

**Comment by matt-peters** [comment]
The training code will utilize the number of GPUs given by `n_gpus`, so seems like a configuration issue on your end.  Common culprits are installing `tensorflow` instead of `tensorflow-gpu`, masking of GPUs with `CUDA_VISIBLE_DEVICES`, problems with CUDA installation, etc.
